src/create_sssom.py

```diff
@@ -54,8 +56,7 @@ def run(input_mappings: str = INPUT_MAPPINGS, input_sssom_config: str = INPUT_CO
     #  move the col removals below (umls) to above
     # - add mapping_justification
     df_hpo_mesh['mapping_justification'] = 'semapv:ManualMappingCuration'
-    write_sssom(df_hpo_mesh, input_sssom_config,
-                OUTPUT_FILE_HPO_MESH.replace('.sssom.tsv', '-non-matches-included.sssom.tsv'))
+    write_sssom(df_hpo_mesh, input_sssom_config, OUTPUT_FILE_HPO_MESH_WITH_NON_MATCHES)
```

Comment:
Is the file `hpo-mesh_non-matches-included.sssom.tsv` still saved to the same location so that any downstream processes that need this file can still access it? 

And was the change in the filename from `...mesh-non...` (with a hyphen) to `...mesh_non...` (with an underscore) intentional? It looks like this is accounted for in all places I can see, but want to confirm.

nit: There are a few levels of directories to visually navigate here with `OUTPUT_FILE_HPO_MESH_WITH_NON_MATCHES` so the readability is a bit difficult. Since `write_sssom` expects `outpath: Union[Path, str]` you may be able to to use `Path` in something like `full_path = Path(base_dir) / sub_dir1 / sub_dir2 / filename`. However, as long as the file is still saved in the same place or any downstream processes know the new location, it is ok to leave this as-is.

Reply:
> Is the file hpo-mesh_non-matches-included.sssom.tsv still saved to the same location so that any downstream processes that need this file can still access it?
It is saved at the same location, yes! I just decided to make a variable representing its path, as opposed to the hacky way I was creating the filename/path beforehand.

FYI though, it is not actually a release artefact, and it is also not used anywhere right now. If I remember correctly, Nico asked me to filter those non-matches out of the original release artefact `hpo-mesh.sssom.tsv`, and I think he gave me the option to save them if I needed. IIRC, I think I wanted to upload them to a google sheet as part of some evaluation process, I don't really remember. So if we want, we probably don't even need to save them, but it's not doing any harm either. The file isn't released and it isn't committed either.

> And was the change in the filename from ...mesh-non... (with a hyphen) to ...mesh_non... (with an underscore) intentional? It looks like this is accounted for in all places I can see, but want to confirm.
Yeah, it was intentional! I figured using a `_` would help separate out the two parts of this filename, the `hpo-mesh`, which is the primary identifier for the matching set, and `non-matches-included`, which is more of a qualifier.
Also I checked and I did indeed rename the path consistently across the codebase (only 2 locations).

> There are a few levels of directories to visually navigate here with OUTPUT_FILE_HPO_MESH_WITH_NON_MATCHES so the readability is a bit difficult. 
Maybe you mean how I have this dir `output/` and I also have `output/release/`. Yeah, this is a new pattern I'm trying. I also have in some other repo I think a setup like `io/input/` and `io/output/...`. I've been wanting to explore these minor tweaks and see which pattern I end up liking better.

>  something like `full_path = Path(base_dir...`
Yeah, I have also been inconsistent in how I'm doing these paths in various repos / places. I am trying to always do the type declaration `Union[str, Path]` though, I think that's good. But I should probably use `Path` instead of `str` whenever possible, too.